cloud-core/src/main/java/cloud/commandframework/arguments/flags/CommandFlag.java

```diff
@@ -94,15 +94,17 @@ private CommandFlag(
      * Get the flag description
      * <p>
      *
+     * @param sender Command sender
      * @return Flag description
      * @deprecated for removal since 1.4.0. Use {@link #getArgumentDescription()} instead.
      */
     @Deprecated
-    public cloud.commandframework.@NonNull Description getDescription() {
+    @SuppressWarnings("unchecked")
+    public cloud.commandframework.@NonNull Description<?> getDescription(final Object sender) {
```

Comment:
we can't break api -- also you lose the sender type here

Reply:
The entire class isn't generic and uses `?` in place of sender types. Would you prefer the class was generified?